pkg/sqldb/postgres.go

```diff
@@ -84,7 +98,22 @@ func (db *PostgresDB) Init(ctx context.Context, host, port, username, password,
 			}
 
 		}
-		initerr = db.RunScript(ctx, createScript)
+		err := db.RunScript(ctx, createScript)
+		switch err.(type) {
+		case nil:
+		case *pq.Error:
+			// The same race condition for creating the database exists for
+			// concurrently creating the tables, in which case the error
+			// is ignored.
+			pqErr := err.(*pq.Error)
+			if pqErr.Code != pqUniqueViolationErrCode {
+				initerr = pqErr
+				return
+			}
+		default:
+			initerr = err
+			return // from the unnamed once.Do function
+		}
```

Comment:
question: when both unique violation codepaths are hit, do the tests still pass?

Reply:
Yes, that's what I see in my local testing